includes/core/classes/class-event-feed.php

```diff
@@ -0,0 +1,298 @@
+				/* translators: %s: Formatted date and time */
+				__( 'Date: %s', 'gatherpress' ),
+				$display_datetime
+			);
+		}
+
+		return $event_info;
+	}
+
+	/**
+	 * Get default event excerpt customization.
+	 *
+	 * @since 1.0.0
+	 *
+	 * @param string $excerpt The current excerpt.
+	 * @return string The customized excerpt.
+	 */
+	public function get_default_event_excerpt( string $excerpt ): string {
+		global $post;
+
+		// Only apply to events.
+		if ( Event::POST_TYPE !== get_post_type( $post ) ) {
+			return $excerpt;
+		}
+
+		$event = new Event( get_the_ID() );
+		$venue = $event->get_venue_information();
+
+		$event_info = $this->get_event_datetime_info( $event );
+
+		// Add venue information.
+		if ( $venue && ! empty( $venue['name'] ) ) {
+			$event_info[] = sprintf(
+				/* translators: %s: Venue name */
+				__( 'Venue: %s', 'gatherpress' ),
+				$venue['name']
+			);
+		}
+
+		// Build the customized excerpt.
+		$custom_excerpt = '';
+
+		if ( ! empty( $event_info ) ) {
+			$custom_excerpt .= '<p><strong>' . implode( ' | ', $event_info ) . '</strong></p>';
+		}
+
+		// Add the original excerpt if it exists and is different from the content.
+		if ( ! empty( $excerpt ) && get_the_content() !== $excerpt ) {
+			$clean_excerpt = wp_strip_all_tags( $excerpt );
+			$clean_excerpt = preg_replace( '/\s+/', ' ', $clean_excerpt ); // Normalize whitespace.
+			$clean_excerpt = trim( $clean_excerpt );
+
+			if ( ! empty( $clean_excerpt ) ) {
+				$custom_excerpt .= '<p>' . $clean_excerpt . '</p>';
+			}
+		}
+
+		return $custom_excerpt;
+	}
+
+	/**
+	 * Get default event content customization.
+	 *
+	 * @since 1.0.0
+	 *
+	 * @param string $content The current content.
+	 * @return string The customized content.
+	 */
+	public function get_default_event_content( string $content ): string {
+		global $post;
+
+		// Only apply to events.
+		if ( Event::POST_TYPE !== get_post_type( $post ) ) {
+			return $content;
+		}
+
+		$event = new Event( get_the_ID() );
+		$venue = $event->get_venue_information();
+
+		$event_info = $this->get_event_datetime_info( $event );
+
+		// Add venue information.
+		if ( $venue && ! empty( $venue['name'] ) ) {
+			$event_info[] = sprintf(
+				/* translators: %s: Venue name */
+				__( 'Venue: %s', 'gatherpress' ),
+				$venue['name']
+			);
+		}
+
+		// Build the customized content.
+		$custom_content = '';
+
+		if ( ! empty( $event_info ) ) {
+			$custom_content .= '<p><strong>' . implode( ' | ', $event_info ) . '</strong></p>';
+		}
+
+		// For RSS feeds, provide a cleaner version of the content.
+		// Strip out complex HTML and keep only essential information.
+		$clean_content = wp_strip_all_tags( $content );
+		$clean_content = preg_replace( '/\s+/', ' ', $clean_content ); // Normalize whitespace.
+		$clean_content = trim( $clean_content );
+
+		if ( ! empty( $clean_content ) ) {
+			$custom_content .= '<p>' . $clean_content . '</p>';
+		}
+
+		// Add comments section.
+		$comments = get_comments(
```

Comment:
What does the normal post feed do with comments? We should do the same thing here.